app/src/main/java/de/tum/in/tumcampusapp/component/ui/chat/activity/ChatRoomsActivity.java

```diff
@@ -155,14 +155,20 @@ private void startJoinRoom() {
 
     @Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-        if (requestCode == 1 && resultCode == RESULT_OK && data != null) {
+        Utils.log("onActivity is this ");
+        if (resultCode == RESULT_OK && data != null) {   //removed requestCode; but maybe inspite of requestCode = 1, we need something else.
```

Comment:
it should be requestCode == JOIN_ROOM_REQUEST_CODE (the constant that is given to the startActivityForResult in the method above)